chart/compass/charts/gateway/templates/oathkeeper-authenticator-rules.yaml

```diff
@@ -0,0 +1,30 @@
+{{- range $authenticatorName, $config := .Values.global.authenticators -}}
+{{if eq $config.enabled true}}
+---
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: "compass-gateway-{{ $authenticatorName }}"
+spec:
+  description: Configuration of oathkeeper for secure endpoint of compass {{ $authenticatorName }} gateway
+  upstream:
+    url: "http://compass-gateway.{{ $.Release.Namespace }}.svc.cluster.local:{{ $.Values.global.gateway.port }}"
+  match:
+    methods: ["GET", "POST", "OPTIONS"]
+    url: <http|https>://compass-gateway-{{ $authenticatorName }}.{{ $.Values.global.ingress.domainName }}<(:(80|443))?>/director/graphql
```

Comment:
perhaps we should have another variable to say what would  be the URL part as the name might be something more verbose than the URL